test/Distributed/Runtime/distributed_actor_protocol_call_resilient_lib.swift

```diff
@@ -52,7 +52,11 @@
 // RUN:     -o %t/a.out
 
 // RUN: %target-codesign %t/a.out
```

Comment:
@ktoso you also need to codesign the dylibs that are using in %target-run

Reply:
Oh, thank you, done now